.github/workflows/reusable-test.yml

```
@@ -0,0 +1,166 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This workflow executes a single test, optionally gathering code coverage and logs.
+
+name: Reusable Test Workflow
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      # The test command to invoke.
+      test_command:
+        required: true
+        type: string
+      # The name of the build job to wait for.
+      build_job:
+        required: true
+        type: string
+      # The name of the build artifact to download.
+      build_artifact:
+        required: true
+        type: string
+      # The environment to run this test on.
+      environment:
+        description: 'Environment to run tests against'
```

```suggestion
        description: 'Environment to run tests on'
```
(wording consistency with line 26)